.github/workflows/CI-DOCS.yml

```
@@ -0,0 +1,86 @@
+        run: |
+          pip install --upgrade "pip>=24.3.1" "setuptools>=75.0" "wheel>=0.44" "build>=1.2.1";
+          pip install -r ./requirements.txt ;
+          pip install -r ./tests/requirements.txt || true ;
+      - name: Pre-Clean
+        id: clean-prep
+        run: make -j1 -f Makefile clean ;
+      - name: Pre-build for Python ${{ matrix.python-version }} on ${{ matrix.os }}
+        run: make -j1 -f Makefile build ;
+        if: ${{ success() }}
+      - name: Generate documentation with py${{ matrix.python-version }} on ${{ matrix.os }}
+        run: make -j1 -f Makefile build-docs 2>&1 >> $GITHUB_STEP_SUMMARY ;
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2069:warning:1:33: To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify)

(shellcheck)